src/test/java/freemarker/core/OutputFormatTest.java

```diff
@@ -322,14 +322,14 @@ public void testAutoEscapingSettingLayers() throws Exception {
     public void testNumericalInterpolation() throws IOException, TemplateException {
         getConfiguration().setRegisteredCustomOutputFormats(Collections.singleton(DummyOutputFormat.INSTANCE));
         assertOutput(
-                "<#ftl outputFormat='dummy'>#{1.5}; #{1.5; m3}; ${'a.b'}",
+                "<#ftl outputFormat='dummy'><#setting locale='en_US'>#{1.5}; #{1.5; m3}; ${'a.b'}",
```

Comment:
That's a good catch, but the locale should be simply set once in the `Configuration`, rather that in each FTL snippet again and again. In fact, the `TemplateTest` should do that automatically, as it's pointless to run tests with a random locale.